prog/weaver/main.go

```diff
@@ -70,6 +78,13 @@ func main() {
 	flag.StringVar(&ipsubnetCIDR, "ipsubnet", "", "subnet to allocate within by default, in CIDR notation")
 	flag.IntVar(&peerCount, "initpeercount", 0, "number of peers in network (for IP address allocation)")
 	flag.StringVar(&apiPath, "api", "unix:///var/run/docker.sock", "Path to Docker API socket")
+
+	flag.StringVar(&dnsDomain, "dns-domain", gossipdns.DefaultDomain, "local dns domain")
+	flag.IntVar(&dnsPort, "dns-port", 53, "port to listen to DNS requests")
+	flag.IntVar(&dnsTTL, "dns-ttl", 30, "TTL for DNS request")
+	flag.IntVar(&dnsUDPbufsize, "dns-udp-bufsize", 4096, "size of udp buffer for fallback dns requests")
+	flag.DurationVar(&dnsClientTimeout, "dns-client-timeout", 5*time.Second, "timeout for fallback dns requests")
```

Comment:
Those option names don't obviously correspond to their description. Can we do better?